### PR TITLE
fix with new e-mail for the broken search feature

### PR DESCRIPTION
**Pull request by biancode**
see issue #6 - from my side issue #3 is also done with the PR

### CODE REVIEW

**Comment by biancode** [comment]
Is this tool at its end of life?

**Comment by biancode** [comment]
You have a sign from me as biancode or do you need a new sign from biancoroyal, please?

**Comment by knolleary** [comment]
Not end of life, just overlooked as it so rarely gets any issues or PRs.

Thanks for bumping - will review etc today.

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,5 @@ build/Release
 
 # Dependency directory
 node_modules
+
+.idea/
```
Comment (dceejay) [inline .gitignore:28]: do we need this ?

File: clean.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```
Comment (dceejay) [inline clean.sh:1]: What is this file for ? don't think we need to add extra files.

File: npm-update.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```
Comment (dceejay) [inline npm-update.sh:1]: likewise - superfluous

File: npm-upgrade.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```
Comment (dceejay) [inline npm-upgrade.sh:1]: and again .. what even is ncu ?

File: .gitignore (hunk 1 above)
Comment (biancode) [reply .gitignore:28]: it is for my JetBrains environment - normally I put it in my repos but wont put it into others

File: clean.sh (hunk 2 above)
Comment (biancode) [reply clean.sh:1]: it is for my JetBrains environment to check shell cmds in files via shellcheck

File: npm-update.sh (hunk 3 above)
Comment (biancode) [reply npm-update.sh:1]: it is for my JetBrains environment to check shell cmds in files via shellcheck

File: npm-upgrade.sh (hunk 4 above)
Comment (biancode) [reply npm-upgrade.sh:1]: npm install -g npm-check-updates = ncu to get automated upgrades to all dependencies of the project - I could set it to be ignored in the .gitignore - helpfully in development to check and/or update dependencies

**Comment by biancode** [comment]
I made an update to the shell scripts to use just sh to be more compatible to other systems.

**Comment by knolleary** [comment]
As a point of principle, PRs should be focused on the issue they are fixing. This PR touches 14 files, most of which is completely unrelated.

Please either provide a PR that fixes the issue in question, or provide a summary of the changes that help us understand what is going on.

**Comment by biancode** [comment]
> As a point of principle, PRs should be focused on the issue they are fixing. This PR touches 14 files, most of which is completely unrelated.
> 
> Please either provide a PR that fixes the issue in question, or provide a summary of the changes that help us understand what is going on.

The source change is focused to fix an issue. The other changes are to get a better development local. (coverage, testing, ...)

**Comment by knolleary** [comment]
> The other changes are to get a better development local.

As I said, these are changes that are unrelated to the fix and shouldn't be part of this PR.

There are also multiple changes to the README and help text output and lots of the dependencies have changed. 

None of the 'better development' changes are documented or explained. As Dave highlighted, one uses `ncu` - but there's no explanation on what that is or how a developer would get ahold of it.

I'm happy to get the search issue resolved, but the other changes are getting in the way of that.

They need to be discussed in their own right, but right now, we can't merge this PR as-is.

**Pull request closed without merging**